go/libkb/config.go

```diff
@@ -414,11 +414,17 @@ func (f JSONConfigFile) GetGpgOptions() []string {
 func (f JSONConfigFile) GetRunMode() (RunMode, error) {
 	var err error
 	var ret RunMode = NoRunMode
-	if s, isSet := f.GetStringAtPath("run-mode"); isSet {
+	if s, isSet := f.GetStringAtPath("run_mode"); isSet {
```

Comment:
Should we change `chat-db` above too? Also any chance this is going to break someone's dev setup?